src/chapters/typescript/functions.rst

```diff
@@ -0,0 +1,115 @@
+Functions in TypeScript
+=======================
+
+When creating functions in TypeScript, we have many of the same options as in JavaScript.
+We can make anonymous functions, give it a different number of parameters and so on.
+However, when working with functions in TypeScript, we have to keep *type* in mind.
+
+Declaring Functions
+-------------------
+
+Named Functions
+^^^^^^^^^^^^^^^
+
+Let's take a look at a TypeScript function declaration.
+
+.. sourcecode:: typescript
+
+   function myFunction(a: number): number {
+      return a*2;
+   }
+
+Here you can see that we provided the type of the parameters and the type of the value that is returned.
+If the function doesn't have a return value, then you use ``void`` for its type.
```

Comment:
similarly, this warrants an example directive of its own. I don't think the idea of something called ``void`` has been introduced yet.

Reply:
Added an example! Thanks!